creusot-contracts/src/model.rs

```diff
@@ -2,14 +2,15 @@ use ::std::{rc::Rc, sync::Arc};
 
 use crate::*;
 
-/// The shallow model of a type is typically used to specify a data
-/// structure. This kind of model is mostly useful for notation purposes,
+/// The view of a type is its logical model as typically used to specify a data
+/// structure. It is typically "shallow", and does not involve the model of
+/// other types contained by the datastructure.
+/// This kind of model is mostly useful for notation purposes,
 /// because this trait is linked to the @ notation of pearlite.
-/// Models of inner types are typically not involved.
-pub trait ShallowModel {
-    type ShallowModelTy;
+pub trait View {
+    type ViewTy;
     #[logic]
-    fn shallow_model(self) -> Self::ShallowModelTy;
+    fn view(self) -> Self::ViewTy;
 }
```

Comment:
Maybe for a future change, but I think it might be good to move `View` to a separate module.